### PR TITLE
Remove connections to virtuoso

### PR DESCRIPTION
**Pull request by Mighantos**
Removes all usages of SSP Virtuoso endpoint.

### CODE REVIEW

File: src/main/java/com/github/sgov/server/dao/VocabularyDao.java [hunk 1]
```diff
@@ -65,4 +69,23 @@ public void clearVocabularyContext(final URI vocabularyContext) {
             throw new PersistenceException(e);
         }
     }
+
+    /**
+     * Returns a set of URIs of vocabularies imported by the vocabulary URI of which is provided.
```
Comment (blcham) [inline src/main/java/com/github/sgov/server/dao/VocabularyDao.java:74]:
```suggestion
     * Returns a set of vocabulary URIs imported by the provided vocabulary URI.
```

File: src/main/java/com/github/sgov/server/service/repository/VocabularyRepositoryService.java [hunk 2]
```diff
@@ -114,7 +96,7 @@ public List<VocabularyDto> getVocabulariesAsContextDtos(String lang) {
             List<VocabularyDto> contexts = new ArrayList<>();
             final SPARQLRepository repo =
                 new SPARQLRepository(IdnUtils.convertUnicodeUrlToAscii(
-                    repositoryConf.getReleaseSparqlEndpointUrl()));
+                    repositoryConf.getUrl()));
```
Comment (blcham) [inline src/main/java/com/github/sgov/server/service/repository/VocabularyRepositoryService.java:99]: Not sure why u decided to rewrite getTransitiveImports() to JOPA but not this part.

File: src/main/java/com/github/sgov/server/service/repository/VocabularyRepositoryService.java [hunk 3]
```diff
@@ -325,9 +307,10 @@ GraphQueryResult loadAttachments(
         final String vocabularyVersion,
         final RepositoryConnection connection) {
         final GraphQuery query = connection
-            .prepareGraphQuery("PREFIX : <"
```
Comment (blcham) [inline src/main/java/com/github/sgov/server/service/repository/VocabularyRepositoryService.java:328]: See comment above.

File: src/main/java/com/github/sgov/server/service/repository/VocabularyRepositoryService.java (hunk 2 above)
Comment (blcham) [reply src/main/java/com/github/sgov/server/service/repository/VocabularyRepositoryService.java:99]: IMHO, we either want to support any SPARQL enpoint that is attached to JOPA or support SPARQL endpoint without JOPA.

**Review by blcham — APPROVED** [review]
would work most likely, but strange implementation to me. See my comments.

File: src/main/java/com/github/sgov/server/service/repository/VocabularyRepositoryService.java (hunk 2 above)
Comment (Mighantos) [reply src/main/java/com/github/sgov/server/service/repository/VocabularyRepositoryService.java:99]: I was trying to make it more readable, but you are right. I will change getTransitiveImports back to SPARQL endpoint without JOPA.

**Pull request merged**